modules/gapi/include/opencv2/gapi/own/mat.hpp

```diff
@@ -205,6 +205,7 @@ namespace cv { namespace gapi { namespace own {
         */
         void create(Size _size, int _type)
         {
+            GAPI_Assert(_size.height >= 0 && _size.width >= 0);
```

Comment:
Why 0 values are included?

Reply:
Because `CV_Assert( s >= 0 )` is used in cv::Mat in `create(...)` in `setSize(...)`.

cv::Mat successfully creates Size{0, 0} mat.

It is a similar check for own::Mat.